### ISSUE TITLE
Email address validation too strict - no international characters allowed

### ISSUE THREAD

**Issue by heslegend** [wiredashio/wiredash-sdk#333]
**Describe the bug**
The email address validation does not allow special international characters like e.g. the german ä, ö, ü.

### Steps to reproduce:
1. Try to submit feedback
2. Enter an email address containing the special character

**Wiredash SDK Info**
_Which SDK are you using?_
1.9.0

_On which platform are you seeing the crash?_
Android, but should be the same on iOS.

**Flutter SDK**
not relevant

**Screenshots**

**Comment by passsy** [comment]
You're right! We need to relax the email validation in the SDK and the backend

Dankeschön!
